pycbc/inference/sampler/ultranest.py

```diff
@@ -52,11 +52,16 @@ class UltranestSampler(BaseSampler):
     ----------
     model : model
         A model from ``pycbc.inference.models``.
+    log_dir : str
+        Folder where files should be stored for resuming (optional).
+    stepsampling : bool
+        If false, uses rejection sampling. If true, uses
+        hit-and-run sampler, which scales better with dimensionality.
     """
     name = "ultranest"
     _io = UltranestFile
 
-    def __init__(self, model, **kwargs):
+    def __init__(self, model, log_dir=None, stepsampling=False, **kwargs):
```

Comment:
Thanks for adding support for stepsampling. In order to set this when running, you'll need to add it to the [list of known options](https://github.com/gwastro/pycbc/blob/6679ae985f26ebaf1a4fbf4d4f34a0bff7257fb5/pycbc/inference/sampler/ultranest.py#L105-L118) in the `from_config` function, so that it can be turned on/off using the configuration file.